src/content/docs/dev/send-tokens/interchain-tokens/developer-guides/link-custom-tokens-deployed-across-multiple-chains-into-interchain-tokens.mdx

```diff
@@ -300,120 +307,158 @@ main().catch((error) => {
 
 ```bash
-Salt: 0x7f5fe989334ad23b77e8da841f672394d021ae83ff0a24afb461ba19d72e3553,
-Transaction Hash: 0x026ee7992de2108ecb83f37119ec84ebed371ff724d38e8fd055cbecde5b77e6,
-Token ID: 0x11780bda6e8238e24cdcd88e9b0088f1ed354c8b03c836f35f9ddb9e2c8be7c5,
-Expected Token Manager Address: 0xad8Cf8FF7BBe82269844bBcbA60bAa8f02781f13,
+Transaction Hash: 0xf860bd8802d046fb0c752c355bc73c87e69f701348420c6cbdf82534b87c4822,
+
 ```
 
-### Store the token ID, Expected Token Manager, and salt value
+### Run the `customInterchainToken.js` script to deploy to Avalanch Fuji
 
-Copy the token ID, Expected Token Manager, and salt value and store them somewhere safe. You will need them to initiate a remote token manager deployment and token transfer in a later step.
+Run the script in your terminal to register and deploy the token, specifying the `avalanche` testnet:
 
-### Check the transaction on the Fantom testnet scanner
+```bash
+FUNCTION_NAME=registerTokenMetadataOnAvalanche npx hardhat run customInterchainToken.js --network avalanche
+```
 
-Check the [Fantom testnet scanner](https://testnet.ftmscan.com/) to see if you have successfully [deployed a token manager](https://testnet.ftmscan.com/tx/0x026ee7992de2108ecb83f37119ec84ebed371ff724d38e8fd055cbecde5b77e6).
+If you see something similar to the following on your console, you have successfully registered your token metadata on AValanche Fuji.
 
-## Deploy a remote token manager on the Polygon testnet
+```bash
+Transaction Hash: 0x026ee7992de2108ecb83f37119ec84ebed371ff724d38e8fd055cbecde5b77e6,
+
+```
+
+### Check the transaction on the BSC testnet scanner
+
+Check the [Fantom testnet scanner](https://testnet.bscscan.com/) to see if you have successfully [registered your token metadata](https://testnet.bscscan.com/tx/0xf860bd8802d046fb0c752c355bc73c87e69f701348420c6cbdf82534b87c4822).
 
-You’ve just successfully deployed a token manager to Fantom, which you are using as your local chain. Now, deploy a token manager remotely to Polygon, which will act as the remote chain in this tutorial. Remember that you can specify any two chains as your local and remote chains.
+### Check the transaction on the Avalanche Fuji testnet scanner
 
-### Estimate gas fees
+Check the [Avalanche Fuji testnet scanner](https://testnet.snowtrace.io/) to see if you have successfully [registered your token metadata](https://testnet.snowtrace.io/tx/0xe3bb64c76637fcd4d9ce9a5bab013fcda95e7040292531230be70daa4f5bd9a1).
 
-In `customInterchainToken.js`, call `estimateGasFee()` from the [AxelarJS SDK](https://www.notion.so/bd238d757a144a069501f0b481488ef3?pvs=21) to estimate the actual cost of deploying your remote Canonical Interchain Token on a remote chain:
+## Register the custom token with the Interchain Token Factory
+
+You've registered your token metadata on BSC and Avalanche Fuji. Now, register the custom token with the Interchain Token Factory on the source chain, which in this example is BSC.
```

Comment:
why does the token need to be registered only on binance but not avalanche? for metadata we needed to register on both chains. Could be a question readers (myself included) have

Reply:
That's a great question! I believe the reason it's not registered on both chains is that, after registering, you only need to link it to the destination chain. Since both chains have metadata registered, the hub is able to recognize it.